BKG/Booking.BLL.Tests/Services/BookingServiceTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿using FluentAssertions;
+
+namespace Booking.BLL.Tests.Services;
+
+public class BookingServiceTest
+{
+    private readonly Mock<IBookingRepository> _bookingRepository;
+    private readonly IBookingService _bookingService;
+    private readonly Mock<IMapper> _mapper;
+
+    public BookingServiceTest()
+    {
+        _bookingRepository = new Mock<IBookingRepository>();
+        _mapper = new Mock<IMapper>();
+        _bookingService = new BookingService(_bookingRepository.Object, _mapper.Object);
+    }
+
+    [Fact]
+    public async Task GetAll_WhenEntitiesExist_ShouldReturnListOfModels_Async()
```

Comment:
What _Async means?
Not needed I suppose

Reply:
All async methods need to have an Async postfix i think

Reply:
No, not needed